acquia/acquia_cms
# ACMS-832: Updated the custom profile readme files and separated acquia_claro theme

**Motivation**
* The Acquia Claro theme needs to be separated.
Fixes #[ACMS-832](https://backlog.acquia.com/browse/ACMS-832)

**Proposed changes**
* Followed the steps mentioned in ticket to incorporate the changes.

**Alternatives considered**
* None

**Testing steps**
* Testing is in progress, will update the steps here once tested. -> Can only be tested once this is merged, check comment in the PR below.

**Merge requirements**
